terraform/modules/oidc/_variable.tf

```diff
@@ -2,24 +2,36 @@ variable "oidc_github_repo" {
   description = "The GitHub repository for OIDC"
   type        = string
   default     = ""
+  validation {
+    condition     = length(var.oidc_github_repo) == 0 || can(regex("^[a-zA-Z0-9-]+/[a-zA-Z0-9-]+$", var.oidc_github_repo))
+    error_message = "oidc_github_repo must be set with 'org/repo' format or blank"
+  }
 }
 
 variable "owner" {
   description = "The owner of the project"
   type        = string
   default     = "skylight"
+  validation {
+    condition     = can(regex("^[[:alnum:]]{1,8}$", var.owner))
+    error_message = "owner must be 8 characters or less, all lowerspace with no special characters or spaces"
```

Comment:
"lowercase"?

Reply:
Hah, I misread your comment. My thought was, yeah, lowercase... 😆 

I'll fix the typo.